src/utils/daos_hdlr.c

```diff
@@ -2566,95 +2576,266 @@ fs_copy(struct file_dfs *src_file_dfs,
 	return rc;
 }
 
+inline void
+set_dm_args_default(struct dm_args *dm)
+{
+	dm->src = NULL;
+	dm->dst = NULL;
+	uuid_clear(dm->src_p_uuid);
+	uuid_clear(dm->src_c_uuid);
+	uuid_clear(dm->dst_p_uuid);
+	uuid_clear(dm->dst_c_uuid);
+	dm->src_poh = DAOS_HDL_INVAL;
+	dm->src_coh = DAOS_HDL_INVAL;
+	dm->dst_poh = DAOS_HDL_INVAL;
+	dm->dst_coh = DAOS_HDL_INVAL;
+	dm->cont_prop_oid = DAOS_PROP_CO_ALLOCED_OID;
+	dm->cont_prop_layout = DAOS_PROP_CO_LAYOUT_TYPE;
+	dm->cont_layout = DAOS_PROP_CO_LAYOUT_UNKOWN;
+	dm->cont_oid = 0;
+
+}
+
+static int
+dm_get_cont_prop(daos_handle_t coh,
+		 char *sysname,
+		 daos_cont_info_t *cont_info,
+		 int size,
+		 uint32_t *dpe_types,
+		 uint64_t *dpe_vals)
+{
+	int                     rc = 0;
+	int                     i = 0;
+	daos_prop_t		*prop = NULL;
+
+	prop = daos_prop_alloc(2);
```

Comment:
@dsikich Should this instead be
```suggestion
	prop = daos_prop_alloc(size);
```